src/app/formConfig.ts

```diff
@@ -44,6 +44,39 @@ export const timer = [
                 ],
                 'default': 'Untitled Course'
             },
+            {
+                'code': 'startDate',
+                'dataType': 'date',
+                'description': 'start Date',
+                'editable': true,
+                'inputType': 'date',
+                'label': 'Start Date',
+                'name': 'Start Date',
+                'placeholder': 'start Date',
+                'renderingHints': {
+                    'class': 'sb-g-col-lg-1 required'
+                },
+                'required': true,
+                'visible': true,
+                'default': '20-10-2021',
+                'validations': [
+                    {
+                        'type': 'minDate',
+                        'value': new Date().toJSON().split('T')[0],
+                        'message': 'Date should not be less than '+new Date().toJSON().split('T')[0]
+                    },
+                    {
+                        'type': 'maxDate',
+                        'value':"2022-01-01",
```

Comment:
`minData` and `maxDate` value should also have to same format as `dateFormat` . Have you verified that?